lib/jetmon.js

```diff
@@ -285,6 +292,34 @@ function getMoreSites() {
 	});
 }
 
+function maybeEndRound() {
+	if ( inRound && 0 === arrObjects.length && arrWorkers.length === freeWorkers.length ) {
+		// Still in the round. No work is queued. All the workers are free. The round has ended.
+		inRound = false;
+
+		var timeSinceStart = new Date().valueOf() - startTime;
+		var timeToNextLoop = ( getRoundDuration() * SECONDS ) - timeSinceStart;
+		var sps = roundSitesCount / timeSinceStart * 1000;
```

Comment:
This `sps` calculation makes more sense to me compared to `sitesCount / global.config.get( 'STATS_UPDATE_INTERVAL_MS' ) * 1000`